python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -603,6 +746,9 @@ def load_assets_from_dbt_project(
         display_raw_sql (Optional[bool]): [Experimental] A flag to indicate if the raw sql associated
             with each model should be included in the asset description. For large projects, setting
             this flag to False is advised to reduce the size of the resulting snapshot.
+        stream_events (Optional[bool]): [Experimental] A flag to indicate if events should be
```

Comment:
Why would someone ever not want this?

Reply:
well dbt versions < 1.4 do not have sufficient metadata attached to their logs in order to produce these events, but I guess we should just do the swapping of behavior ourselves (checking the dbt version)